src/styles/utils/animations.scss

```diff
@@ -0,0 +1,9 @@
+@keyframes move-hand {
+  from {
+    transform: translate(-50%, -100%) rotate(0);
```

Comment:
You can use here `turn` value for `transform: rotate`([rotate MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate)) for simplicity.